### PR TITLE
issue #193: Profile page creation 

### PR DESCRIPTION
**Pull request by VSciFlight**
# Description
Added an empty space page for profile to be manifested in.
It is not shown on the sidebar menu and accessible only by reaching the link of /profile.

## screenshots
image

### CODE REVIEW

File: src/App.tsx [hunk 1]
```diff
@@ -76,6 +77,10 @@ const PAGES = [
     label: TEXT_KEYS.about_title,
     key: '/about',
   },
+  // {
+  //   label: TEXT_KEYS.profile_page,
+  //   key: '/profile',
+  // },
```
Comment (NoamGaash) [inline src/App.tsx:83]: please remove,
you can add a different array for invisible pages

**Review by NoamGaash — APPROVED** [review]
Nice start!
Please resolve conflicts and comments :muscle:

File: src/App.tsx (hunk 1 above)
Comment (VSciFlight) [reply src/App.tsx]: conflicts were resolved

**Comment by NoamGaash** [comment]
please see the linter and playwright test failures

**Comment by VSciFlight** [comment]
Strange thing because it worked fine before the commit. 
afterwards it got messy so I rebranched.

Fixed it now, suppose to work just fine and just upgraded it
closing this request and opening new relevant one.

**Pull request closed without merging**